unit-tests-jest/numbers.test.ts

```diff
@@ -0,0 +1,57 @@
+import { evenNumbers, toDollars, divideBy, multiplyBy } from './numbers';
+
+// how to make test fails ? expect(evenNumbers([4, 6, 1, 23, 4, 8])).
+
+// individual test
+test('Returns a new array that contains only the even numbers in the given array.', () => {
+  expect(evenNumbers([4, 6, 1, 23, 4, 8])).toEqual([4, 6, 4, 8]);
```

Comment:
It's best to create a new test for each of these cases. That lets you fix things one at a time and more easily figure out what is wrong with the code.